packages/lesswrong/components/posts/PostsPage/PostsTopSequencesNav.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react'
 import { registerComponent, Components } from 'meteor/vulcan:core';
+import { withRouter } from 'react-router';
 
-const PostsTopSequencesNav = ({classes, post, sequenceId}) => {
+const PostsTopSequencesNav = ({post, sequenceId, router}) => {
   const { SequencesNavigation, CollectionsNavigation } = Components
   const canonicalCollectionSlug = post.canonicalCollectionSlug;
   const title = getNavTitle(post)
   const titleUrl = getNavTitleUrl(post)
 
-  if (sequenceId && !canonicalCollectionSlug) {
+  const isSequenceRoute = () => {
+    if ((router && router.routes && router.routes.length > 1) && router.routes[1]) {
```

Comment:
`routes` should be available directly in the props without having to go through `router`. 

Also: you should check whether any of the routes match the "sequencesPost" name, not just the second one. This also simplifies some of the other code.